verification/curator-service/api/src/controllers/sources.ts

```diff
@@ -3,10 +3,36 @@ import { Source } from '../model/source';
 
 /**
  * List the sources.
+ * Response will contain {sources: [list of sources]}
+ * and potentially another nextPage: <num> if more results are available.
+ * Default values of 10 for limit and 1 for page is used.
  */
 export const list = async (req: Request, res: Response): Promise<void> => {
-    const docs = await Source.find({});
-    res.json(docs);
+    const page = Number(req.query.page) || 1;
+    const limit = Number(req.query.limit) || 10;
+    if (page < 1) {
+        res.status(422).json('page must be > 0');
+        return;
+    }
+    if (limit < 1) {
+        res.status(422).json('limit must be > 0');
+        return;
+    }
+    const docs = await Source.find({})
+        .skip(limit * page - limit)
```

Comment:
hyper-nit: limit * (page - 1) is slightly clearer IMO

Reply:
done